whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -258,33 +259,13 @@ private Connector createInsecureConnector(final Server server, final HttpConfigu
 
     /**
      * Use the DoSFilter from Jetty for rate limiting: https://www.eclipse.org/jetty/documentation/current/dos-filter.html.
-     * See {@link WhoisDoSFilter} for the customisations added.
-     * @return the rate limiting filter
+     * See {@link WhoisDoSFilter} or {@link WhoisDoSFilter} for the customisations added.
      * @throws JmxException if anything goes wrong JMX wise
      * @throws JMException if anything goes wrong JMX wise
      */
-    private FilterHolder createDosFilter() throws JmxException, JMException {
-        WhoisDoSFilter dosFilter = new WhoisDoSFilter();
-        FilterHolder holder = new FilterHolder(dosFilter);
-        holder.setName("DoSFilter");
-
-        if (!dosFilterEnabled) {
-            LOGGER.info("DoSFilter is *not* enabled");
-        }
-        holder.setInitParameter("enabled", Boolean.toString(dosFilterEnabled));
-        holder.setInitParameter("maxRequestsPerSec", "50");
-        holder.setInitParameter("maxRequestMs", "" + 10 * 60 * 1_000); // high default, 10 minutes
-        holder.setInitParameter("delayMs", "-1"); // reject requests over threshold
-        holder.setInitParameter("remotePort", "false");
-        holder.setInitParameter("trackSessions", "false");
-        holder.setInitParameter("insertHeaders", "false");
-        holder.setInitParameter("ipWhitelist", trustedIpRanges);
-
-        if (!ManagementFactory.getPlatformMBeanServer().isRegistered(dosFilterMBeanName)) {
-            ManagementFactory.getPlatformMBeanServer().registerMBean(new ObjectMBean(dosFilter), dosFilterMBeanName);
```

Comment:
We do not want to register them as JMX bean ?

Reply:
I will check the Jira. I guess that the idea here is to register the beans. Why would we need a JMX for this?
Maybe if we want to modify the whiteList at any moment? Let me check.
Thanks for pointing out this

Reply:
Talked via slack. We will create a new filter for black list IPs and that is the one that need to have the JMX call and not this one